### ISSUE TITLE
Can't find variable Pc on release apk

### ISSUE THREAD

**Issue by Blapi** [facebook/react-native#13039]
### Description

I am working on an Android App using react-native and I wanted to test my application on a release apk but after following the steps there (https://facebook.github.io/react-native/docs/signed-apk-android.html), when I launch the app on my phone, it crashes as soon as it opens and I've got the following error in Android Studio logcat :

```
03-20 14:16:14.484 17860-17894/? E/ReactNativeJS: Can't find variable: Pc
03-20 14:16:14.488 17860-17894/? E/ReactNativeJS: Module AppRegistry is not a registered callable module (calling runApplication)
03-20 14:16:14.502 17860-17895/? E/AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
                                                   Process: com.jogabo, PID: 17860
                                                   com.facebook.react.common.JavascriptException: Can't find variable: Pc, stack:
                                                   <unknown>@480:4814
                                                   <unknown>@480:4879
                                                   <unknown>@482:22906
                                                   <unknown>@482:22919
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@473:44
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@472:79
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@470:374
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@469:458
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@468:88
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@366:253
                                                   n@2:557
                                                   i@2:348
                                                   t@2:210
                                                   <unknown>@12:38
                                                   n@2:557
                                                   i@2:278
                                                   t@2:210
                                                   global code@595:9
                                                   
                                                       at com.facebook.react.modules.core.ExceptionsManagerModule.showOrThrowError(ExceptionsManagerModule.java:99)
                                                       at com.facebook.react.modules.core.ExceptionsManagerModule.reportFatalException(ExceptionsManagerModule.java:83)
                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                       at com.facebook.react.bridge.BaseJavaModule$JavaMethod.invoke(BaseJavaModule.java:345)
                                                       at com.facebook.react.cxxbridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:136)
                                                       at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
                                                       at android.os.Handler.handleCallback(Handler.java:751)
                                                       at android.os.Handler.dispatchMessage(Handler.java:95)
                                                       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:31)
                                                       at android.os.Looper.loop(Looper.java:154)
                                                       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:196)
                                                       at java.lang.Thread.run(Thread.java:761)
03-20 14:16:14.508 4342-16308/? W/ActivityManager:   Force finishing activity com.jogabo/.MainActivity
```
Weird thing is that this is working well when I'm using a debug apk

### Additional Information

* React Native version: ^0.42.0
* Platform: Android
* Operating System: Linux (Ubuntu 16.04)
* Dev tools: Nexus 5X (Android 7.1.1) - android:minSdkVersion="16" android:targetSdkVersion="22"

**Comment by Blapi** [comment]
Fixed this when downgrading on React native 0.40

**Comment by iiitmahesh** [comment]
same problem with  

```
"react": "16.0.0-alpha.3",
 "react-native": "0.43.1",
```
